src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -31,8 +33,19 @@ public BookDto getBookById(@PathVariable Long id) {
     }
 
     @PostMapping
-    @ResponseStatus(HttpStatus.CREATED)
     public BookDto createBook(@RequestBody CreateBookRequestDto bookRequestDto) {
         return bookService.save(bookRequestDto);
     }
+
+    @PutMapping("/{id}")
```

Comment:
Add response status accepted

Reply:
I am not sure it is appropriate, simple `200 - OK` should be enough. The `202 - Accepted` status code means that the request has been accepted for processing, but the processing has not been finished yet. The request may or may not be completed when the processing eventually takes place. 
But you should also ask Andrii what he thinks about it

Reply:
I assume that Andrii has agreed to my comment, next time please add a comment on what was the resolution to that question, so that I do not have to guess